src/autoload.ts

```diff
@@ -222,7 +222,7 @@ export function autoload(viewer?, callback?) {
                         var modelName = viewerdiv.dataset[datatypes[i]];
                         UI.setModelTitle(modelName);
                     }
-                    i += 1;
+                    i = (i as number) + 1;
```

Comment:
why does this need an explicit cast?

Reply:
Removed the explicit cast and redeclared `i` to be `let i = 0` in Line 215

Had added the cast to solve this error,
<img width="680" alt="Screenshot 2023-07-20 at 9 18 05 PM" src="https://github.com/3dmol/3Dmol.js/assets/23498248/1e0e9910-459f-4842-9ef3-8850545c7ede">